src/feature/dirauth/voting_schedule.c

```diff
@@ -20,50 +20,6 @@
  * Vote scheduling
```

Comment:
Perhaps it's now time to update the file-level `\brief` comment of `voting_schedule.c` to indicate that this file should only be used by dirauth now?

Reply:
good idea.